pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,257 @@
+        you should put the operator at the beginning of the line and not to the end of the line.
+    </p>
+    <pre><code class="language-python"># No
+result = (
+    1 +
+    2 *
+    3
+)
+
+# Yes
+result = (
+    1
+    + 2
+    * 3
+)</code></pre>
+    <p>If you compare against <code>None</code>, you should use <code>is</code> and <code>is not</code>, but not compare equality.</p>
+    <pre><code class="language-python"># No
+if variable == None:
+    print("Variable is None")
+
+# Yes
+if variable is None:
+    print("Variable is None")</code></pre>
+    <p>
+        You should prefer using <code>&lt;item one&gt; is not &lt;item two&gt;</code> over <code>not &lt;item one&gt; is &lt;item two&gt;</code>.
+        Using second makes it harder to understand the expression.
+    </p>
+    <pre><code class="language-python"># No
+if not variable is None:
+    print("Variable is not None")
+
+# Yes - it is much easier to read and understand this than previous
+if variable is not None:
+    print("Variable is not None")</code></pre>
+
+    <h3>Imports</h3>
+    <p>Imports should be at top of the file, the only things that should be before them are module comments and docstrings.</p>
+    <p>You shouldn't import multiple modules in one line, but give each module import its own line instead.</p>
+    <pre><code class="language-python"># No
+import pathlib, os
+
+# Yes
+import os
+import pathlib</code></pre>
+    <p>Wildcard imports should be avoided in most cases. They make unclear where what comes from.</p>
+    <pre><code class="language-python"># No
+from pathlib import *
+
+# Yes
+from pathlib import Path</code></pre>
+    <p>You should use <b><a href="https://pycqa.github.io/isort/">isort</a></b> imports order specification, what means:</p>
+    <ul>
+        <li>
+            <b>Group by type:</b> order of import types should be: <code>__future__</code> imports, standard library imports,
+            third-party library imports, and finally project imports.
+        </li>
+        <li>
+            <b>Group by import method:</b> inside each group, first should come imports in format <code>import &lt;package&gt;</code>
+            and after them <code>from &lt;package&gt; import &lt;items&gt;</code>.
+        </li>
+        <li>
+            <b>Order imports alphabetically:</b> inside each import method group, imports should be ordered by package names.
+        </li>
+        <li>
+            <b>Order individual import items by type and alphabetically:</b> in <code>from &lt;package&gt; import &lt;items&gt;</code> format,
+            <code>&lt;items&gt;</code> should be ordered alphabetically, starting with bare module imports.
+        </li>
+    </ul>
+
+    <h3>Comments</h3>
+    <p>
+        Comments are really important because they help everyone understand what code does.
+        But as important as having comments is keeping them up-to-date.
+        Out-to-date, wrong comments confuse readers.
+    </p>
```

Comment:
Rewriting this and adding a bit of text to explain what comments should aim to do. 

```suggestion
    <p>
        Comments are really important because they help everyone understand what code does.
        In general, comments should explain <i>why</i> you are doing something if it's not obvious.
        You should aim to write code that makes it obvious what it is doing and you can use the comments to explain why and provide some context.
        <br>
        Keep in mind that just as important as having comments, is making sure they stay up to date.
        Out-of-date and incorrect comments confuse readers of your code (including future you).
    </p>
```